README.md

```diff
@@ -1,8 +1,313 @@
+
+* Run the client
+
+  Executing the client with `--help` option will show the usage instuctions
+```bash
+    > bin/fuseml --help
+bin/fuseml is a command line client for the FuseML API.
+
+Usage:
+    fuseml [-host HOST][-url URL][-timeout SECONDS][-verbose|-v] SERVICE ENDPOINT [flags]
+
+    -host HOST:  server host (dev). valid values: dev, prod
+    -url URL:    specify service URL overriding host URL (http://localhost:8080)
+    -timeout:    maximum number of seconds to wait for response (30)
+    -verbose|-v: print request and response details (false)
+
+Commands:
+    application (list|register|get|delete)
+    codeset (list|register|get|delete)
+    runnable (list|register|get)
+    workflow (list|register|get|assign|list-runs)
+
+Additional help:
+    bin/fuseml SERVICE [ENDPOINT] --help
+
+Example:
+    bin/fuseml application list --type "predictor" --workflow "mlflow-sklearn-e2e"
+    bin/fuseml codeset list --project "mlflow-project-01" --label "mlflow"
+    bin/fuseml runnable list --id "ml-trainer-123" --kind "trainer" --labels '{
+          "function": "predict|train",
+          "library": "pytorch"
+       }'
+    bin/fuseml workflow list --name "workflow"
+```
+
+  Instead of providing `-url` argument to each command call, save the service URL into `FUSEML_SERVER_URL` enviornment variable and export it. Use this command to fill the variable with the URL of the `fuseml-core` server service that is installed in your Kubernetes cluster:
+
+  ```bash
+  export FUSEML_SERVER_URL=$(kubectl get VirtualService -n fuseml-core fuseml-core -o jsonpath="{.spec.hosts[0]}")
+  ```
+
+  If neither `FUSEML_SERVER_URL` nor `-url` value is set, client tries to connect to the server running locally at your machine.
+
+
+  From the usage output you can see that with the client you can operate with various FuseML artefacts.
+
+  * Codesets describe the code of your ML application, for example MLflow project. They are currently implemented as git repositories.
+
+    Create new codeset by calling `bin/fuseml codeset register` command. use the `--help` option to get the description of the command usage.
+
+    Example:
+    ```bash
+    bin/fuseml codeset register --name "test" --project "mlflow-project-01" --location "/tmp/mlflow/mlflow-01"
+    ```
+
+    `--location` argument points to the directory on your machine where your ML application code is located.
+
+    After registering, use
+    ```
+    bin/fuseml codeset list
+    ```
+    command to list available codesets and check the value of "URL" in the output of your newly registered codeset. Use this value to git clone the code into another directory. Now you can work in this directory just like with any other project saved in git. Assuming you have configured a workflow assignment (see parts about workflows), any time you commit and push new changes to your code, a worklflow run will be executed and you can expect updated application as a result.
```

Comment:
suggestion: `...Assuming that you assigned a workflow to this codeset (see workflows section), every time you push new code changes a new workflow run will be created. `